src/library/Uno.Material/Styles/Controls/Divider.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:converters="using:Uno.Material.Converters"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:toolkit="using:Uno.UI.Toolkit">
+
+	<ResourceDictionary.MergedDictionaries>
```

Comment:
Do not include colors, otherwise it will override the color palette.

Reply:
It was crashing when I didn't include them. I will try to figure out another solution

Reply:
Well it's not crashing anymore. Maybe it was something else. I've removed it